NuGet.config

```diff
@@ -12,5 +12,6 @@
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vs-editor-mac" value="https://www.myget.org/F/vs-editor/api/v3/index.json" />
+    <add key="SemanticPR" value="https://devdiv.pkgs.visualstudio.com/_packaging/VSIDE-RealSigned-Release%40Local/nuget/v3/index.json" />
```

Comment:
Odd, why aren't new versions of packages we've been getting from the above feeds not in those feeds?

Reply:
At the time I sent this the PR in question had not been merged. Once I get confirmation on a version that has it I'll be removing that feed.